src/pages/receipts/packagesInTransit.js

```diff
@@ -345,33 +324,33 @@ const groupScannedByArr = (bagsArr, fieldToConceptIdMapping) => {
             let groupNames = [];
             for (let j = 0; j < Object.keys(bag).length; j++) {
               // First name and last name exist
-              if(bag[Object.keys(bag)[j]][scannedByFirstName] && bag[Object.keys(bag)[j]][scannedByLastName]){
+                if(bag[Object.keys(bag)[j]][scannedByFirstName] && bag[Object.keys(bag)[j]][scannedByLastName]){
```

Comment:
Please avoid repeated computing Object.keys(bag), by saving and referring its value.